### DISCUSSION TITLE
In <Avatar/> component Why are there some very tiny borders in this place?

### DISCUSSION THREAD

**Discussion by geekskai** [chakra-ui/chakra-ui#4303] [Q&A]
Why are there some very tiny borders around the Avatar?

this is [demo](https://codesandbox.io/s/condescending-bogdan-tuse5?file=/src/App.js)

When I add a little style to this Avatar component, when I click to switch, the tiny border on it disappears!

![image](https://user-images.githubusercontent.com/37830362/123797809-27331100-d919-11eb-8105-c794afd6da02.png)

But if I refresh the page many times, this very subtle border may disappear. Look at the picture below. Does anyone know what the problem is? ?

![image](https://user-images.githubusercontent.com/37830362/123797869-387c1d80-d919-11eb-99a9-2015ab7f0002.png)

**Comment by kelilipan** [comment]
Because the fallback component has a background color, idk if the image size is 1px smaller or because of something else.
Just add bgColor='transparent' to remove the small border.
https://codesandbox.io/s/hardcore-sanderson-qetbc?file=/src/App.js

```jsx
<Avatar
  size="2xl"
  //this remove fallback bg
  bgColor="transparent"
  color="black" //make sure alias is readable
  name="wisesa"
  src="https://www.wisesa.dev/favicon/android-icon-96x96.png"
/>
```

**Reply by geekskai** [reply]
At the beginning, I tried you to do this, but when the name and src are not available, the default avatar should be online. If you follow your scheme, the default avatar will be completely transparent and nothing can be seen!

[https://codesandbox.io/s/vigorous-rgb-qcxpy?file=/src/App.js:499-531](https://codesandbox.io/s/vigorous-rgb-qcxpy?file=/src/App.js:499-531)



``` html
 <Avatar
   size="2xl"
   color="black"
   // The default avatar will be overwritten, and nothing will be seen!
   bgColor="transparent"
   // name="wisesa"
   // src="https://www.wisesa.dev/404"
  />
```
